Technical-Steering-Committee/logging.md

```diff
@@ -0,0 +1,56 @@
+# Message Management
+
+## How do we log
+
+### Message ID Components (message prefixes)
+
+ZWE - Zowe, This should be used as default for all Zowe messages. 
+
+### Zowe Projects
+
+There are server and client side components that are logging separately as such we may have duplication of logging for Server side and Client side. 
+
+#### Server Side
+
+A - API Mediation Layer
+D - Zowe App Framework
+S - Zowe System Services
+
+#### Client Side 
+
+C - Zowe CLI
+E - Zowe VSCode Explorer
```

Comment:
In practice, does this mean Zowe Explorer messages like job submission failed would be presented as a `ZWEExxxxE` message?  The benefit being that we could distinguish this type of error from an extension error or some other error message?